packages/react-components/src/icons/src/EmbeddedIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import { Children, ReactElement, ReactNode } from "react";
+import { augmentElement, createSizeAdapter } from "../../shared";
+
+export interface EmbeddedIconProps {
+    /**
+     * An icon can vary in size.
+     */
+    size?: "2xs" | "xs" | "sm" | "md" | "lg" | "xl" | "inherit";
```

Comment:
I might be confused but I think you told me you need to use the () syntax for the enum to render correctly in SB

Reply:
I was mistaken